app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -68,19 +67,18 @@ If you’re not sure of the status of your visa, check:
 * [UK visa support for Ukrainian nationals](https://www.gov.uk/guidance/support-for-family-members-of-british-nationals-in-ukraine-and-ukrainian-nationals-in-ukraine-and-the-uk)
 * [Apply to stay in the UK under the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
```

Comment:
@laurahermionetennant92 there's a line of content not showing here that says You are eligible which could be changed to You're eligible